src/core/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ if(ENABLE_SECURITY)
     $<BUILD_INTERFACE:$<TARGET_PROPERTY:security_api,INTERFACE_INCLUDE_DIRECTORIES>>)
 endif()
 
-if(iceoryx_binding_c_FOUND)
```

Comment:
It is indeed better to test for `ENABLE_SHM`, but perhaps it would be a good idea to then do this consistently?
```
rg iceoryx_binding_c_FOUND ../src 
../src/security/core/tests/CMakeLists.txt
117:if(iceoryx_binding_c_FOUND)

../src/core/ddsc/tests/CMakeLists.txt
107:if(iceoryx_binding_c_FOUND)
141:if(iceoryx_binding_c_FOUND)
152:if(iceoryx_binding_c_FOUND AND NOT WIN32)

../src/core/xtests/rhc_torture/CMakeLists.txt
21:if(iceoryx_binding_c_FOUND)

../src/core/ddsi/CMakeLists.txt
181:if(iceoryx_binding_c_FOUND)
```

Reply:
I wasn't aware it was this bad... I'll replace these too.

Reply:
Done